src/parser/nodes.js

```diff
@@ -40,20 +40,20 @@ module.exports = {
     }
   },
 
-  DialogOptionNode: class extends Shortcut {
+  DialogShortcutNode: class extends Shortcut {
     constructor(text, content, lineNo) {
       super();
-      this.type = 'DialogOptionNode';
+      this.type = 'DialogShortcutNode';
```

Comment:
renaming these result nodes will break bondagejs in projects using it (it is a breaking api change) - would it be possible to keep their name as it was?

@hylyh are we ok to change the api so drastically?

This matters in the case where a wrapper using bondagejs is checking the node's type

Reply:
I couldn't keep things straight in my head when _shortcuts_ were named _options_ and _options_ were named _links_. I'd prefer to keep the names aligned with what those objects are called in yarnspinner docs. It makes maintenance and bug fixing much easier.